config/config-features.yaml

```diff
@@ -0,0 +1 @@
+core/configmaps/features.yaml
```

Comment:
just checking: this isn't the same as what the commit/issue title says?

Reply:
Thank you. Updated the commit/issue title.